### PR TITLE
Search Improvements

### PR DESCRIPTION
**Pull request by SaraVieira**
In a user test it was pointed we can't order the search and it makes sense to do so

This PR adds order by likes and views

This PR also:

* Changes the cards in search for the new cards we have in the explore page
* Removes how long the search took as that is not important information for the user
* Changes the API of wide sandbox to have the prop `selectSandbox` instead of `pickSandbox` to make it more generic
* Adds common script
* Moves some .css files to gobal js styles so we can use the theme

Design
https://www.figma.com/file/akq4c3BMhkW5EHipeIBEbKmf/Search-Page?node-id=0%3A1

### CODE REVIEW

**Comment by SaraVieira** [comment]
cc @Haroenv

**Comment by Haroenv** [comment]
The sorting of "relevance" and "views" seems very similar to me, was a different sort required for this? 

I'm seeing some flashes when I change index, but I don't see any particular reason why in the code by a quick glance, maybe it already was there before



Also, it seems like the search is pretty hidden with the new design, I can't seem to find any link towards it anymore, I had to guess the URL.

Finally there's a small other thing (maybe only on Safari), and that's the search icon isn't correctly positioned: 

Screenshot 2019-03-21 at 15 56 36

**Comment by SaraVieira** [comment]
Good points!

I deleted the views one as that is basically the default one and also added a link to the search in the header

The other two:

The flickering seems to only happen on safari :/

I'm looking at the CSS thing now

**Comment by CompuIves** [comment]
The bottom part of the card (with description) seems a bit big to me, does it have different styles compared to `SandboxCard` in `common`?

**Comment by SaraVieira** [comment]
@CompuIves Nop, itmay be because most of them don't have a description

File: packages/app/src/app/pages/Search/Results/index.js [hunk 1]
```diff
@@ -2,17 +2,30 @@ import React from 'react';
...
 import { Container } from './elements';
 
 function Results() {
   return (
     <Container>
       <ResultInfo />
-      <Hits hitComponent={SandboxCard} />
+      <Hits
+        hitComponent={({ hit }) => (
+          <WideSandbox
```
Comment (CompuIves) [inline packages/app/src/app/pages/Search/Results/index.js]: It does seem a bit weird, after checking it seems like the description of the sandbox in search is ~150px and in explore ~130px height

File: packages/common/src/components/WideSandbox/index.tsx [hunk 2]
```diff
@@ -34,7 +34,7 @@ export type Props = {
   small?: boolean;
   defaultHeight?: number;
   noMargin?: boolean;
-  pickSandbox: (
+  selectSandbox: (
```
Comment (CompuIves) [inline packages/common/src/components/WideSandbox/index.tsx]: Nice!!

File: packages/app/src/app/pages/Search/Filters/index.js [hunk 3]
```diff
@@ -1,12 +1,22 @@
...
+import { ALGOLIA_DEFAULT_INDEX } from 'common/lib/utils/config';
 
 import Filter from './Filter';
+import Sort from './Filter/Sort';
 import { Container } from './elements';
 
 function Filters() {
   return (
     <Container>
+      <Sort
```
Comment (CompuIves) [inline packages/app/src/app/pages/Search/Filters/index.js]: This is nice! Maybe we can also re-use 

image

in a later version.

File: package.json [hunk 4]
```diff
@@ -75,7 +76,8 @@
     "gulp": "^3.9.1",
     "lerna": "^2.5.1",
     "lint-staged": "^5.0.0",
-    "prettier": "^1.8.2"
+    "prettier": "^1.8.2",
+    "typescript": "^3.3.4000"
```
Comment (CompuIves) [inline package.json]: Could this one be `^3.3.1`? That way we ensure that we have a single typescript version in the whole repo.

Comment (SaraVieira) [reply package.json]: Fixed

**Pull request merged**